File: Stack/Stack.java
/*
 * Stack follows LIFO: Last In First Out
 */

package DataStructure;

public class Stack {
	// Create Stack
	int size = 3;
	int top = -1;
	int[] stack = new int[size];
	
	 // Elements adding method
	 void push(int x) {
		if(top < size-1) {
			top += 1;
			stack[top] = x;
			System.out.println("Element added: "+x);
		}
		else {
			System.out.println("Ops! There is no space.");
		}
	}
	
	// Element deleting method from top
	void pop() {
		if(top >= 0) {
			int value = stack[top];
			top -= 1;
			System.out.println("Element removed: "+value);
		}
		else {
			System.out.println("Ops! Empty Stack.");
		}
	}
	
	// Showing Elements from top
	void peek() {
		if(top >= 0) {
			System.out.println("Top Element: "+stack[top]);
		}
		else {
			System.out.println("Ops! Empty Stack.");
		}
	}
	
	/////////////// Main Function
	public static void main(String[] args){
		Stack stack = new Stack();
		stack.push(40);
		stack.push(20);
		stack.push(90);
		stack.push(50);
		stack.peek();
		stack.pop();
		stack.peek();
		stack.pop();
		stack.pop();
		stack.peek();
	}
}

File: Stack/Stack.js
/////////// Basic Structure of stack
let stack = [];
let initialSize = stack.length;
let max = 5;
let count = 0;
//// Element adding
function push(x){
    if(initialSize < max){
        stack[initialSize] = x;
        console.log('Element added: '+x);
        initialSize += 1;
    }
    else{
        console.log('Ops! There is no space.');
    }
}

//// Element remove from top
function pop(){
    if(initialSize > 0){
        console.log('Element remove from top.');
        initialSize--;
    }
    else{
        console.log('Ops! Empty Stack.')
    }
}

//// Top element showing
function peek(){
    if(initialSize > 0){
        console.log('Top element of Stack: '+stack[initialSize-1]);
    }
    else{
        console.log('Ops! Empty Stack');
    }
}

///////////////// Main function
push(10);
push(20);
push(30);
push(40);
push(50);

peek();
pop();
peek()
push(90)
peek();
console.log(stack);